src/lib/ThumbnailsSidebar.js

```diff
@@ -254,6 +254,7 @@ class ThumbnailsSidebar {
 
         if (pageNum === this.currentPage) {
             thumbnailEl.classList.add(CLASS_BOX_PREVIEW_THUMBNAIL_IS_SELECTED);
+            thumbnailEl.setAttribute('aria-current', true);
```

Comment:
Does this value get updated as the user changes between pages?

Reply:
@jstoffan you are right. I added and removed the aria-current when user changes between pages.